spyder-custom-layout/spyder_custom_layout/spyder/plugin.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2021, Spyder Bot
+#
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
+"""
+Spyder Custom Layout Plugin.
+"""
+
+# Third-party imports
+from qtpy.QtGui import QIcon
+
+# Spyder imports
+from spyder.api.plugins import Plugins, SpyderDockablePlugin
+from spyder.api.translations import get_translation
+
+# Local imports
+from spyder_custom_layout.spyder.confpage import SpyderCustomLayoutConfigPage
+from spyder_custom_layout.spyder.widgets import SpyderCustomLayoutWidget
+from spyder_custom_layout.spyder.api import CustomLayout
+
+_ = get_translation("spyder_custom_layout.spyder")
+
+
+class SpyderCustomLayout(SpyderDockablePlugin):
```

Comment:
Shouldn't this be a `SpyderPluginV2`? Since it only adds a layout, I think that's the class we should inherit from here, right?

Reply:
It is a `SpyderDockablePlugin` to also ahow the `TABIFY` property